src/extensions/DatatypeTrait.php

```diff
@@ -37,7 +37,7 @@ private function getDatatypeFromString($value) {
             return 'integer';
         }
 
-        if (preg_match('/^[+-]?([0-9]*[.])?([0-9]|[.][0-9])+$/', $value)) {
+        if (preg_match('/(^[+-]?$)|(^[+-]?[0-9]+([,.][0-9])?[0-9]*(e[+-]?[0-9]+)?$)/', $value)) {
```

Comment:
Thanks. Is not needed for us, but correct.

Reply:
Actually, note to myself: I have to remove the comma, as PHP natively only supports the point.

Reply:
But we need the comma.

Reply:
Ok, it'll stay.